app/design/adminhtml/default/default/layout/edge/pages.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layout>
+    <adminhtml_cms_page_type_index>
+        <reference name="content">
+            <block type="pages/adminhtml_cms_page_type" name="type"></block>
```

Comment:
these can be declared kind of like xhtml
<block type="pages/adminhtml_cms_page_type" name="type"/>